internal/reorder/cmds.go

```diff
@@ -3,9 +3,10 @@ package reorder
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/aviator-co/av/internal/meta"
 	"io"
 
+	"github.com/aviator-co/av/internal/meta"
+
```

Comment:
why the newlines? 🤔 

also I ran the `pre-commit` on all the files, wonder why this wasn't caught.

Reply:
I'm actually not sure...